asm_to_pil/Cargo.toml

```diff
@@ -1,15 +1,16 @@
 [package]
-name = "asm_to_pil"
+name = "powdr-asm_to_pil"
```

Comment:
I think the main thing we need to decide is the name convention of crates with multiple words. Do we use `powdr-asm_to_pil` or `powdr-asm-to-pil`?

Reply:
IMO it's better to use the same separator